opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraphBar.jsx

```diff
@@ -865,6 +871,18 @@ class InvestigationGraphBar extends Component {
                           />
                         </>
                       )}
+                      <Tooltip title={t('Restore the state of the graphic before the last expansion')}>
+                        <span>
+                          <IconButton
+                            color="primary"
+                            disabled={isRollBackToLastPreExpansionStateEnabled}
```

Comment:
not sure to understand why we disable the restore button if  `isRollBackToLastPreExpansionStateEnabled` is true (or maybe it should be named isRollBackToLastPreExpansionStateDisabled ?)